packages/ui/src/fields/Point/index.tsx

```diff
@@ -27,7 +27,7 @@ export type PointFieldProps = {
   width?: string
 } & FormFieldBase
 
-export const _PointField: React.FC<PointFieldProps> = (props) => {
+export const PointFieldComponent: React.FC<PointFieldProps> = (props) => {
```

Comment:
Same comment as before, might want to remove the export keyword here if not used anywhere else